packages/host/app/components/operator-mode/host-submode.gts

```diff
@@ -43,14 +43,20 @@ export default class HostSubmode extends Component<HostSubmodeSignature> {
   @tracked isPublishRealmModalOpen = false;
   @tracked isPublishingRealmPopoverOpen = false;
   @tracked isOpenSitePopoverOpen = false;
+  @tracked publishError: string | null = null;
+  @tracked publishErrors: Map<string, string> = new Map();
```

Comment:
Is it possible to combine `publishError` into `publishErrors` somehow? It seems clumsy to me to have two of these passed around everywhere and reset.